Misc/NEWS.d/next/Core and Builtins/2022-12-09-13-18-42.gh-issue-100146.xLVKg0.rst

```diff
@@ -0,0 +1,4 @@
+Improve ``BUILD_LIST`` opcode so that it works similarly to the
+``BUILD_TUPLE`` opcode, by stealing references from the stack rather than
```

Comment:
```suggestion
Improve :opcode:`BUILD_LIST` so that it works similarly to 
:opcode:`BUILD_TUPLE`, by stealing references from the stack rather than
```